src/lib.rs

```diff
@@ -1708,6 +1709,44 @@ impl RtcConfig {
         self.reordering_size_video
     }
 
+    /// Sets the number of packets kept for answering resends.
+    pub fn set_resend_size_audio(mut self, size: usize) -> Self {
+        self.resend_size_audio = size;
+        self
+    }
+
+    /// Returns the setting for audio resend size.
+    ///
+    /// ```
+    /// # use str0m::Rtc;
+    /// let config = Rtc::builder();
+    ///
+    /// // Defaults to 50.
+    /// assert_eq!(config.resend_size_audio(), 50);
+    /// ```
+    pub fn resend_size_audio(&self) -> usize {
+        self.resend_size_audio
```

Comment:
what happens if I pick 0?

Reply:
Setting it too low will probably just break. This configures a size of a ring buffer that is used for outgoing packets.
If you can guarantee a perfect 1 in 1 out (write/poll), you can maybe get away with 1.

For sanity I think something like 10 would feel better.

Reply:
That would leave you room in case you end up doing write write write, poll poll poll

Reply:
Maybe I should document this.